src/MacroExpander.js

```diff
@@ -234,11 +253,13 @@ export default class MacroExpander implements MacroContextInterface {
 
     /**
      * Returns the expanded macro as a reversed array of tokens and a macro
-     * argument count.
-     * Caches macro expansions for those that were defined simple TeX strings.
+     * argument count.  Or returns `null` if no such macro.
      */
-    _getExpansion(name: string): MacroExpansion {
-        const definition = this.macros[name];
+    _getExpansion(name: string): ?MacroExpansion {
+        const definition = this.namespace.getMacro(name);
```

Comment:
It would be nice to have a convenience function `getMacro` directly on `MacroExpander` since we do `.namespace.getMacro` in a couple of places.

Reply:
The new `.macros.get(...)` is probably good now.